R/importWaterML1.r

```diff
@@ -97,356 +95,194 @@
 #' tzURL <- constructNWISURL("04027000", c("00300","63680"), "2011-11-05", "2011-11-07","uv")
 #' tzIssue <- importWaterML1(tzURL, TRUE, "America/Chicago")
 #'
-#' 
+#' #raw XML
+#' url <- constructNWISURL(service = 'dv', siteNumber = '02319300', parameterCd = "00060", 
+#'                          startDate = "2014-01-01", endDate = "2014-01-01")
+#' raw <- content(GET(url), as = 'raw')
+#' rawParsed <- importWaterML1(raw)
 #' }
 #' filePath <- system.file("extdata", package="dataRetrieval")
 #' fileName <- "WaterML1Example.xml"
 #' fullPath <- file.path(filePath, fileName)
-#' imporFile <- importWaterML1(fullPath,TRUE)
+#' importFile <- importWaterML1(fullPath,TRUE)
 #'
-importWaterML1 <- function(obs_url,asDateTime=FALSE, tz=""){
-  
-  if(file.exists(obs_url)){
-    rawData <- obs_url
-    returnedDoc <- xmlTreeParse(rawData, getDTD = FALSE, useInternalNodes = TRUE)
+
+importWaterML1 <- function(input,asDateTime=FALSE, tz=""){
```

Comment:
I don't think it's a concern that we'll break someone's script (I don't think anyone's really using it). The only thing might be to keep it consistent with `importWaterML1` so someone could conceivably swap out the functions without much hassle.